### PR TITLE
{vis}[foss/2019b] SimpleElastix v1.1.0 w/ Python 3.7.4 + gtest 1.10.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2414.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6ba16571a70a2f40d6cac4ee0833113c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/36d17037e3bfc0b13080fae6d3499117 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3163.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/b6e5b5590e335ebcc9ad5322e4134863 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/123e3cfee19cc13e90069c5d7b2234cb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso-3 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/254f5a83f70d62d5f83c06f7bf615065 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3117.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/6f910fb275bd8457b21eb4e3dc879b03 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/cb84983ae7596133f9108a7a92924b76 for a full test report.

File: easybuild/easyconfigs/s/SimpleElastix/SimpleElastix-1.1.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+local_itk_configopts += "-DJPEG_LIBRARY=$EBROOTLIBJPEGMINTURBO/lib64/libjpeg.%s " % SHLIB_EXT
+local_itk_configopts += "-DITK_USE_SYSTEM_PNG=ON -DPNG_PNG_INCLUDE_DIR=$EBROOTLIBPNG/include "
+local_itk_configopts += "-DPNG_LIBRARY=$EBROOTLIBPNG/lib/libpng.%s " % SHLIB_EXT
+local_itk_configopts += "-DITK_USE_SYSTEM_TIFF=ON -DTIFF_INCLUDE_DIR=$EBROOTLIBTIFF/include "
+local_itk_configopts += "-DTIFF_LIBRARY=$EBROOTLIBTIFF/lib/libtiff.%s " % SHLIB_EXT
+local_itk_configopts += "-DITK_USE_SYSTEM_ZLIB=ON -DZLIB_INCLUDE_DIR=$EBROOTZLIB/include "
+local_itk_configopts += "-DZLIB_LIBRARY=$EBROOTZLIB/lib/libz.%s " % SHLIB_EXT
+
+local_elastix_configopts = "-DCMAKE_BUILD_TYPE=Release -DELASTIX_BUILD_EXECUTABLE=OFF "
+local_elastix_configopts += "-DITK_DIR=%(builddir)s/ITK-5.0.1/easybuild_obj/"
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/SimpleElastix/SimpleElastix-1.1.0-foss-2019b-Python-3.7.4.eb]: consider using local variables for the versions of ITK and elastix, to make it easier to update later (they are used in multiple places)

Comment (boegel) [reply easybuild/easyconfigs/s/SimpleElastix/SimpleElastix-1.1.0-foss-2019b-Python-3.7.4.eb]: fixed in e9774d4

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3117.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/fbc6f8df026237ee77de4e597e270229 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso-3 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/4fa8baa17eb82539a57a6b7445ab6a5f for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**